en/features_mc/throw_launch.md

```diff
@@ -0,0 +1,87 @@
+# Throw Launch (Multicopter)
+
+:::warning Experimental
+This feature was introduced after PX4 v1.14.
+
+- It has not yet been broadly tested on different vehicle configurations or scenarios. various platforms and in different scenarios.
+- The majority of testing has been done in position mode.
+  Other modes should also work.
+
+:::
+
+This feature allows a multicopter to be started by arming it from a fixed position and then throwing it into the air.
+The vehicle then turns on the motors and operates according to its current mode.
+
+The vehicle detects that it has been thrown based on reaching a certain speed and altitude (5m/s and 2m by default, respectively) and then starts the motors at the apex of the throw, once it determines that it has started to fall.
+Note that unlike in other configurations, when throw launch is enabled the propellers should not spin until the launch is detected ( even when the vehicle is armed).
```

Comment:
Should we also mention that with throw mode enabled, it repeats the arming sound until the vehicle is thrown (or disarmed)?